website/versioned_docs/version-0.9.3/installation.mdx

```diff
@@ -593,3 +599,26 @@ The quickest way to access your dashboard is by setting up a port forward:
 kubectl port-forward svc/ww-gitops-weave-gitops -n flux-system 9001:9001
 ```
 Then, [open the dashboard](http://localhost:9001/).
+
+## gitops CLI
+
+Weave GitOps includes a command-line interface to help users create and manage resources.
+
+The `gitops` CLI is currently supported on Mac (x86 and Arm), and Linux - including Windows Subsystem for Linux (WSL).
+
+Windows support is a [planned enhancement](https://github.com/weaveworks/weave-gitops/issues/663).
```

Comment:
Can we strike this out?  Who knows when we will actually get to it.

Reply:
Assuming its fine to leave, we can always remove in a subsequent update 👍

Reply:
It's fine, you can leave it for now.